app/scripts/lib/derived-keys.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
> I'd prefer to talk about and name the keys about what they "are" (i.e., relier keys), rather than how
> they're implemented (i.e., "derived keys"). 

Funnily enough this file started out named "relier-keys.js" and I changed it; will put it back